.github/workflows/update-website.yml

```diff
@@ -36,6 +37,8 @@ jobs:
           GIT_SSH_COMMAND="ssh -i ./book_publish_key -o UserKnownHostsFile=./known_hosts" git clone git@github.com:asterinas/book.git temp_book
           cd temp_book
           git checkout --orphan newBranch
+          git rm -rf .
```

Comment:
Usually, `git rm -rf <target_dir>` removes the `target_dir`. So taking at its face value, the command `git rm -rf .` looks like it deleting the `.` directory, which happens to be the project root. But obviously, deleting the project root is not what we want. What we want is to delete everything under the project root.

So to improve clarity, I suggest using the following command instead.

```
git rm -rf *
```

It should be equivalent to the original one, although I didn't test it.

Reply:
It is equivalent in this CI.  And these two commands have some slight differences that `git rm -rf *` would keep the files and directories start with `.` while `git rm -rf .` only keeps `.git`. 

There are no `.` files under `book` and `mdbook` won't generate them, so I will replace it with `git rm -rf *`.